assets/common/components/delete_endpoint/spec.yaml

```diff
@@ -8,9 +8,7 @@ is_deterministic: True
 display_name: Delete Endpoint
 description:
   Deletes an endpoint resource.
-
 environment: azureml://registries/azureml/environments/python-sdk-v2/versions/15
-
```

Comment:
why are we removing empty lines? Please revert it, if not needed.

Reply:
reverted